examples/cslc_s1_sample_runconfig-v2.0.0-er.2.0.yaml

```diff
@@ -0,0 +1,174 @@
+# Sample RunConfig for use with the CSLC-S1 PGE v2.0.0-er.2.0
+# This RunConfig should require minimal changes in order to be used with the
```

Comment:
I agree, great job spelling out what each field is and how it is used.  I believe people using these files will be thankful for your care and effort.  Perhaps this level of documentation will become a standard.